tests/unit/commands/package/test_command.py

```diff
@@ -17,6 +17,7 @@ def setUp(self):
         self.metadata = {"abc": "def"}
         self.region = None
         self.profile = None
+        self.resolve_s3 = False
```

Comment:
Fine as a default, but can we add unit tests where this is set to true, and where we stub out the act of actually setting up the bucket?

Especially important here as our unit tests didn't notice that the basic case would never work (you can't set this param because s3_bucket is required).